app/ogShare/route.tsx

```diff
@@ -0,0 +1,30 @@
+import { ImageResponse } from "next/server";
+// App router includes @vercel/og.
+// No need to install it.
+
+export const runtime = "edge";
+
+export async function GET() {
+  const image = await fetch(
+    new URL("../opengraph-image-share.jpg", import.meta.url),
+  ).then((res) => res.arrayBuffer());
+
+  return new ImageResponse(
+    (
+      <div
+        style={{
+          display: "flex",
+          background: "#f6f6f6",
+          width: "100%",
+          height: "100%",
+          flexDirection: "column",
+          justifyContent: "center",
+          alignItems: "center",
+        }}
+      >
+        {/* eslint-disable-next-line */}
+        <img style={{ width: "100%", height: "100%" }} src={image as any} />
```

Comment:
For the share image I mentioned that you should be customizing the text within the OG -> i.e. automatically fill out the query and generate SOQL

Reply:
yeah mb got a lil confused will do